Apply patch at https://github.com/pytorch/pytorch/pull/128159

It was causing the error seen here https://github.com/pytorch/pytorch/issues/127637


Index: pytorch/torch/utils/_triton.py
===================================================================
--- pytorch.orig/torch/utils/_triton.py	2024-05-03 14:37:03.959311601 -0500
+++ pytorch/torch/utils/_triton.py	2024-06-26 14:46:04.562956833 -0500
@@ -100,4 +100,6 @@
 
     backend_hash = triton_backend_hash()
     key = f"{triton_key()}-{backend_hash}"
-    return hashlib.sha256(key.encode("utf-8")).hexdigest()
+
+    # Hash is upper case so that it can't contain any Python keywords.
+    return hashlib.sha256(key.encode("utf-8")).hexdigest().upper()
